src/events/interactionCreate.ts

```diff
@@ -165,3 +145,63 @@ async function handleInteraction(interaction: CommandInteraction): Promise<void>
 
 	return await command.execute(context);
 }
+
+async function handleButtonInteraction(
+	context: InteractionContext,
+	interaction: ButtonInteraction
+): Promise<void> {
+	logger.debug(`User ${logUser(interaction.user)} pressed button: '${interaction.customId}'`);
+
+	const button = allButtons.get(interaction.customId);
+	if (!button) {
+		logger.warn(`Received request to execute unknown button with id '${interaction.customId}'`);
+		return;
+	}
```

Comment:
This was based on a similar pattern in `handleCommandInteraction`. It might be good to just flag this for now. Avg has made a change to this in #63 that would be a good pattern to follow, but that needs to be merged first, or if this gets merged first, the error handling can be added on his branch. I'll put a TODO on it.

Reply:
#63 only needs one more review

Reply:
Merged.

Reply:
Conflict resolution time 😭

Reply:
:excellent:

Reply:
Luckily this ended up being a lot less hairy than I expected heh